onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3792,12 +3808,12 @@ def aten_native_channel_shuffle(self: TensorType, groups: int) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_native_dropout(
-    input: TensorType, p: float, train: Optional[bool]
-) -> tuple[TensorType, TensorType]:
+@torch_op("aten::native_dropout")
+def aten_native_dropout(input: TTensor, p: float, train: bool = True) -> Tuple[TTensor, BOOL]:
```

Comment:
Input type? It needs to match what drop out can take (tensor(bfloat16), tensor(double), tensor(float), tensor(float16)) which is 

```suggestion
def aten_native_dropout(input: TFloatOrBFloat16, p: float, train: bool = True) -> Tuple[TFloatOrBFloat16, BOOL]:
```

Reply:
yes, u r right